packages/@aws-cdk/aws-codebuild/test/codebuild.test.ts

```diff
@@ -1028,6 +1028,93 @@ describe('secondary sources', () => {
   });
 });
 
+describe('sources with customised build status configuration', () => {
+    test('github', () => {
+        const context = "My custom CodeBuild worker!"
+        const stack = new cdk.Stack();
+        const source = codebuild.Source.gitHub(
+            { owner: 'awslabs', repo: 'aws-cdk', buildStatusContext: context }
+        );
+
+        new codebuild.Project(stack, 'MyProject', {
+          source,
+        });
+        Template.fromStack(stack).hasResourceProperties(
+            'AWS::CodeBuild::Project', {
+                'Source': {
+                    'buildStatusConfig': {
+                        'context': context,
+                    }
+                }
+            }
+        )
+    })
+    test('github enterprise', () => {
+        const context = "My custom CodeBuild worker!"
+        const stack = new cdk.Stack();
+        const source = codebuild.Source.gitHubEnterprise({ 
+            httpsCloneUrl: 'url',
+            buildStatusContext: context
```

Comment:
Missing trailing comma:

```suggestion
            buildStatusContext: context,
```